src/views/Calendar.vue

```diff
@@ -131,6 +131,10 @@ import eventLimitText from '../fullcalendar/eventLimitText.js'
 import windowResize from '../fullcalendar/windowResize.js'
 import dayRender from '../fullcalendar/dayRender.js'
 import { loadState } from '@nextcloud/initial-state'
+import {
+	showWarning,
+} from '@nextcloud/dialogs'
+import '@nextcloud/dialogs/styles/toast.scss'
```

Comment:
Why does this one needs to import styles as well?

Reply:
According to https://github.com/nextcloud/nextcloud-dialogs you have to import the styles and Calendar.vue is the root component, so it will always load the styles here.

All the other components are children of Calendar.vue